config.json

```diff
@@ -1,4 +1,6 @@
 {
+  "multipleResponseEnable": true,
```

Comment:
Update the [README.md](https://github.com/rafaelcoelho/jsproxy/blob/master/README.md) about those new configurations.